src/miner_restart_sup.erl

```diff
@@ -144,6 +144,12 @@ init(_Opts) ->
                 []
         end,
 
+    Telemetry = case blockchain_utils:get_boolean_os_env_var("SEND_MINER_TELEMETRY", false)
```

Comment:
use the config.src setup instead of this approach?

Reply:
OK

Reply:
i think only the validator profile has config in the src templates at the moment so we’d need to migrate other profiles to that style of config to configure for this telemetry export

Reply:
This is primarily intended to be for validators, so I think it's probably ok...